tests/integration/nm/ethtool_test.py

```diff
@@ -0,0 +1,38 @@
+## SPDX-License-Identifier: LGPL-2.1-or-later
```

Comment:
The python linter is complaining.. please use a single `#`. Thanks!

Reply:
Resolved. Thanks.